### DISCUSSION TITLE
Could you provide a guide on how to install the localify patch on the mumu android emulator?

### DISCUSSION THREAD

**Discussion by ToastyBuns3939** [chinosk6/gakuen-imas-localify#1] [Q&A] [answered]
I can't seem to get magisk and LSPosed to install properly on the emulator.

**Comment by chinosk6** [comment]
This is the official Chinese tutorial of mumu.
https://mumu.163.com/help/20240202/35044_1136675.html

**Reply by chinosk6** [reply, marked as answer]
You can use `R65A24840-kitsune (26404)` this version of [Kitsune Mask](https://github.com/HuskyDG/magisk-files)
